packages/clerk-js/src/core/clerk.ts

```diff
@@ -722,16 +722,16 @@ export class Clerk implements ClerkInterface {
     //   undefined, then wait for beforeEmit to complete before emitting the new session.
     //   When undefined, neither SignedIn nor SignedOut renders, which avoids flickers or
     //   automatic reloading when reloading shouldn't be happening.
-    const beforeUnloadTracker = createBeforeUnloadTracker();
+    const beforeUnloadTracker = this.#options.standardBrowser ? createBeforeUnloadTracker() : undefined;
```

Comment:
Alternatively, have the factory function accept an argument to indicate whether or not it's a web browser environment.